pkgs/build-support/fetchmavenartifact/default.nix

```diff
@@ -0,0 +1,57 @@
+# Adaptation of the MIT-licensed work on `sbt2nix` done by Charles O'Farrell
+
+{ fetchurl, stdenv }:
+let
+  defaultRepos = [
```

Comment:
I'm not sure about the best way to achieve that. Maybe add another argument next to `repo` and `repos` called `addRepos`? I think the current behavior of `repo` and `repos` is desirable (i.e., preventing lookup in the default maven repos), but what you suggest is also desirable. 

Another alternative would be to take a parameter `modRepos :: [Repo] -> [Repo]`, but that would make calling this function a bit more awkward.

Any ideas?